src/main/java/uk/gov/hmcts/darts/arm/service/impl/AbstractUnstructuredToArmProcessor.java

```diff
@@ -185,19 +181,11 @@ protected boolean copyRawDataToArm(ExternalObjectDirectoryEntity unstructuredExt
                 armExternalObjectDirectory.setLastModifiedBy(userIdentity.getUserAccount());
                 externalObjectDirectoryRepository.saveAndFlush(armExternalObjectDirectory);
             }
-        } catch (BlobStorageException e) {
-            if (e.getStatusCode() == BLOB_ALREADY_EXISTS_STATUS_CODE) {
-                log.info("BLOB raw data already exists {}", e.getMessage());
-            } else {
-                log.error("Failed to move BLOB data for file {} due to {}", unstructuredExternalObjectDirectory.getExternalLocation(), e.getMessage());
-                recoveryAction.run();
-                return false;
-            }
```

Comment:
AzCopy will overwrite the files at the destination if they already exist, it won't throw an exception so this is not needed